Gemfile

```diff
@@ -1,5 +1,3 @@
 source 'https://rubygems.org'
 
```

Comment:
- [ ] You should Use RSpec for unit testing, kindly add it to the gemfile so others can use your code out of the box

Reply:
@Ismailco Thank you for your suggestions. So you mean to simply write these two lines inside the file:
gem 'rubocop', '>= 1.0', '< 2.0'
gem "rspec"

Reply:
@NatiG25, Yes